tests/util.go

```diff
@@ -171,9 +179,12 @@ func NewEnvFromMap(state *state.StateDB, envValues map[string]string, exeValues
 	env.gasLimit = common.Big(envValues["currentGasLimit"])
 	env.Gas = new(big.Int)
 
+	env.evm = vm.New(env, nil)
```

Comment:
Note that this change causes an unexpected regression in the VM benchmarks because initialising the VM now does more work:

```text
~/d/e/s/g/e/go-ethereum >> ~/develop/eth/bin/benchstat old.txt new.txt
name                  old time/op  new time/op  delta
StateCall1024-8       12.6µs ± 1%  19.7µs ± 1%  +55.89%  (p=0.000 n=20+20)
VmAckermann32Tests-8  8.10ms ± 1%  8.22ms ± 1%   +1.53%  (p=0.000 n=19+17)
VmFibonacci16Tests-8  48.5ms ± 1%  48.8ms ± 1%   +0.77%  (p=0.000 n=20+19)
```